### DISCUSSION TITLE
Introduction:  Martin Wollny

### DISCUSSION THREAD

**Discussion by martinprogrammer** [life-itself/community#912] [Introductions]
I'm interested in modelling future, in particular, imagining scenarios and interactions between people trying to solve common problems and work towards a brighter future.  

I'm particularly sensitive to contradictions and paradoxes, and I see them everywhere.  I'd like to work towards designing and implementing future social structures that would address the shortcomings of our current reality.

I believe that progressive growth and the alignment of common goals should be packaged as an ideology that should have both political and commercial aspects.

I would love to see people getting together in close-knit interest based communities and explore the limits of their co-operation and inter-dependability.

I believe that like minded people should be working on augmenting collective power that will appear as a power on the outside as well.

Glad to have found this group.

**Comment by rufuspollock** [comment]
@martinprogrammer a warm welcome 👋 

If you have any questions please feel free to start asking them - hearing them and dialoging responses to them helps us as well as you 😄

> I'm particularly sensitive to contradictions and paradoxes, and I see them everywhere. I'd like to work towards designing and implementing future social structures that would address the shortcomings of our current reality.
> 
> I believe that progressive growth and the alignment of common goals should be packaged as an ideology that should have both political and commercial aspects.

Amazing. A big 👍 to these sentiments.

> I would love to see people getting together in close-knit interest based communities and explore the limits of their co-operation and inter-dependability.
>
> I believe that like minded people should be working on augmenting collective power that will appear as a power on the outside as well.

👍👍
